froggah.py

```diff
@@ -108,13 +106,22 @@
             if row%2 == 0:
                 x = -mov
                 for i in range(cars[row]):
-                    screen.blit(car, (x % screenx, carrow[row]), area=(carinrowbw[i], 0, carwidth, carwidth))
-                    x += distance[row]
+                    if -carwidth <= x < 0:
+                        screen.blit(car, (x, carrow[row]), area=(carinrowbw[i], 0, carwidth, carwidth))
+                        x += distance[row]
+                    else:
+                        screen.blit(car, (x % (screenx + carwidth), carrow[row]), area=(carinrowbw[i], 0, carwidth, carwidth))
+                        x += distance[row]

```

Comment:
i had to use the if statement to have the animation separately, because the images "x" starts in the left of the image and the x is negative when the animation starts